project/SharedForBuild.scala

```diff
@@ -10,9 +10,9 @@ object SharedForBuild extends Build {
   import com.nitro.build._
   import PublishHelpers._
 
-  lazy val semver = SemanticVersion(0, 3, 7, isSnapshot = false)
+  lazy val semver = SemanticVersion(0, 4, 0, isSnapshot = true)
```

Comment:
I think we want the version to _not_ be a snapshot: we want to publish this as `0.4.0`, right?